### PR TITLE
New server plugin: slack_notifications.

### PR DESCRIPTION
**Pull request by buzzdeee**
An easy plugin to send notifications about visits and credentials
to a given slack channel via Slack Webhook.

The pushbullet_notifications plugin was taken as an example
on how to get this to work.

### CODE REVIEW

File: server/slack_notifications.py [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+	def on_kp_db_event(self, sender, targets, session):
+		for event in targets:
+			message = db_manager.get_row_by_id(session, db_models.Message, event.message_id)
+
+			if sender == 'visits':
+				message = "New visit from {0} for campaign '{1}'".format(message.target_email, message.campaign.name)
+			elif sender == 'credentials':
+				message = "<!here> New credentials received from {0} for campaign '{1}'".format(message.target_email, message.campaign.name)
+			else:
+				return
```
Comment (zeroSteiner) [inline server/slack_notifications.py:65]: It looks like this should actually be a continue statement so the rest of the targets are processed but a message isn't sent if the sender is not visits or credentials.

Comment (buzzdeee) [reply server/slack_notifications.py:65]: Hi,

this very same logic is copied from pusbutton_notifications.py.
As I understand the logic, in on_server_initialized() I listen for 'visits' and 'credentials',
so in on_kp_db_event() the else: case can't be hit.

If you prefer a continue, I could change it, however then, I think for consistency, the pushbutton_notifications.py should be changed as well.

Comment (zeroSteiner) [reply server/slack_notifications.py:65]: Yeah, you're right. It looks like I misread the code at first. It makes sense now.

**Comment by zeroSteiner** [comment]
Landed, thanks alot @buzzdeee this is a fantastic addition to notification plugins.

**Pull request merged**